src/Data/Order/OrderDataClass.ts

```diff
@@ -189,23 +154,18 @@ export const Order = provideLocalStorageDataClass('Order', {
       keyword: 'Zentral-Schmieranlage, Metallbau, MAX/40',
       number: 'P-11-007605',
       type: 'PSA_ORD_STY_NEW',
-      typeLocalized: '',
       customer: 'Maier Schmiertechnik GmbH (München)',
       commercialAgent: '052601BEBCEC39C8E040A8C00D0107AC',
       technicalAgent: 'D38D7FBFA277856DE030A8C02A010460',
       salesPartner: '',
       status: 'PSA_PRO_ORD_CLS',
-      statusLocalized: 'order completed',
       mainStatus: 'PSA_SAP_MAI_QUO_ORD_PRP',
-      mainStatusLocalized: 'Evaluation',
       open: false,
       quoteAt: null,
       orderAt: '2024-01-02T23:00:00Z',
       deliveryAt: '2023-07-23T13:35:38Z',
       termsOfDelivery: 'PSA_TOD_10',
```

Comment:
This is probably outdated. See https://edit.scrivito.com/d0a154d76edf2a7bd991fc658e700a1d~https://dave-localize-order.scrivito-portal-app.pages.dev/order-details-75f5feaca1004671?order_id=F5F3792034404D40B41A843B61DC60D1&_scrivito_workspace_id=i6c8d84ddc5950c2&_scrivito_display_mode=diff

Reply:
Pisa still answers with the following data (e.g. `GET /order/F5F3792034404D40B41A843B61DC60D1`):

```json
    "termsOfDelivery": "PSA_TOD_10",
    "termsOfDeliveryLocalized": "",
```

Please clarify.

Reply:
Clarified.